src/me/StevenLawson/TotalFreedomMod/Commands/Command_trail.java

```diff
@@ -18,16 +18,42 @@
 import org.bukkit.plugin.RegisteredListener;
 
 @CommandPermissions(level = AdminLevel.SUPER, source = SourceType.ONLY_IN_GAME)
-@CommandParameters(description = "Pretty rainbow trails.", usage = "/<command> [off]")
+@CommandParameters(description = "Pretty rainbow trails.", usage = "/<command> [off | glass]")
```

Comment:
@JeromSar Ahh, My bad. I never scrolled along. I had mistaken this as a new separate command instead of an addition to the current command. My bad.